frontend/app/.server/domain/multi-channel/search-api-faker.ts

```diff
@@ -14,7 +14,7 @@ function generateFakeHitListResult(seed: number): HitListResult {
     dayOfBirth: faker.number.int({ min: 1, max: 28 }),
     parentSurname: faker.person.lastName(),
     partialSIN: faker.string.numeric(3),
-    score: faker.number.int({ min: 0, max: 100 }),
+    score: faker.number.float({ min: 0, max: 10, fractionDigits: 1 }),
```

Comment:
Fake scrore between 0.8 and 1? Then you multiply by
```suggestion
    score: faker.number.float({ min: 0.8, max: 1),
```

Then you can format it like this:

```ts
function formatToPercentage(num: number, lang: Language = "en") {
  return new Intl.NumberFormat(`${lang}-CA`, { style: 'percent', maximumFractionDigits: 2 }).format(num);
}
```

English Lang

![image](https://github.com/user-attachments/assets/178bc67c-5aeb-4fc8-a688-c429364649e1)

French Lang

![image](https://github.com/user-attachments/assets/08f8e1a8-5c95-4303-9ab2-f6f87d27ab5a)